client/js/store/modules/gameList.js

```diff
@@ -57,12 +58,13 @@ export default {
         });
       });
     },
-    requestCreateGame(context, newGameName) {
+    requestCreateGame(context, { gameName, ranked }) {
```

Comment:
Same thing here, does ranked need to be defaulted? If not, does it make more sense to fast fail if it isn't passed?